src/VisualStudio/IntegrationTest/IntegrationTestBuild.slnf

```diff
@@ -0,0 +1,97 @@
+{
+  "solution": {
+    "path": "..\\..\\..\\Roslyn.sln",
+    "projects": [
+      "src\\Compilers\\CSharp\\CSharpAnalyzerDriver\\CSharpAnalyzerDriver.shproj",
```

Comment:
list came from VS - unloaded everything then loaded all the integration test projects and VSIXs plus their dependencies.